### PR TITLE
Add environment documentation

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/dbb85ff5a558413bc93ceeb39ac76001ff28b3d7/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-88e0de6f`

File: content/sre/gitops/environments.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Mandatory Environments
+
+There are two mandatory environments per tenant:
+
+### Build
+
+Build environment contains all tekton pipeline configurations/resources like *pipeline,eventlistener,pipelinrun etc*. This environment is used for running pipelines of tenant applications.
+
+### Preview
```
Comment (rasheedamir) [inline content/sre/gitops/environments.md]: Isn't this for shared dependencies?

**Comment by stakater-user** [comment]
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/c05b4c4e84b5ce13d57273b789223a3457b0b567/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-89762c9b`

**Review by rasheedamir — DISMISSED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-aed945dd`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-4ec64061`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-b385fc9b`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-e6b2dd64`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-95ebddf3`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-72b8164a`

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-4906a3ad`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-bef2d12c`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-200-4847a5fb`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.142`

**Pull request merged**